test/graphql/customer.test.js

```diff
@@ -82,6 +82,87 @@ describe('Query.customer', () => {
   })
 })
 
+describe.only('Query.customer.authorisedBusinesses', () => {
+  before(mockServer.start)
+  after(mockServer.stop)
+
+  it('should return customer authorisedBusinesses', async () => {
+    const result = await graphql({
+      source: `#graphql
+        query TestCustomerAuthorisedBusinesses($id: ID!) {
+          customer(id: $id) {
+            authorisedBusinesses {
+              id
+              name
+              sbi
+              roles
+              privileges
+            }
+          }
+        }
+      `,
+      variableValues: {
+        id: '5841879'
+      },
+      schema,
+      contextValue: context()
+    })
+
+    deepEqual(result, {
```

Comment:
once you have moved the sample data to the fixture you can use that import to assert the response like https://github.com/DEFRA/ffc-customer-registry-api/blob/main/test/graphql/business.test.js#L81